### DISCUSSION TITLE
[Feature request] 480p and 30 FPS Mode

### DISCUSSION THREAD

**Discussion by AntonIXO** [ChristopherKlay/StadiaEnhanced#108] [Ideas & Requests]
Hi, 
Sometimes I am playing with bad internet.
So can you add  less quality modes?
Thanks

**Comment by ChristopherKlay** [comment]
I sadly currently can't.

Limiting the stream to 1080p or lower results in some issues with Stadias settings sometimes switching to 720p as a max resolution available, even if you switch to a higher resolution afterwards.
